keyboards/ploopyco/madromys/readme.md

```diff
@@ -0,0 +1,31 @@
+
+# Ploopyco Madromys Trackball
+
+It's a DIY, QMK-powered trackball!
+
+* Keyboard Maintainer: [PloopyCo](https://github.com/ploopyco)
+* Hardware Supported: RP2040
+* Hardware Availability: [Store](https://ploopy.co), [GitHub](https://github.com/ploopyco)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb ploopyco/madromys -km default
+
+# Building Firmware
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
```

Comment:
I strongly feel this should have a "how to reset to bootloader mode" section.

Reply:
To clarify, it should have a "how to enter bootloader" section.

Reply:
Fixed in [cb93979](https://github.com/qmk/qmk_firmware/pull/21989/commits/cb93979b8cc002c13a68ea800ad72e9e94b44458).

I added the above text, and added a link to a photo hosted on the Ploopy servers.